tests/testAssessmentOrdering.js

```diff
@@ -0,0 +1,123 @@
+const util = require('util');
+const assert = require('chai').assert;
+const { step } = require('mocha-steps');
+
+const config = require('../lib/config');
+const sqldb = require('../prairielib/sql-db');
+const sqlLoader = require('../prairielib/sql-loader');
+const sqlEquiv = sqlLoader.loadSqlEquiv(__filename);
+
+const helperServer = require('./helperServer');
+const helperClient = require('./helperClient');
+
+describe('Course with assessments grouped by unit vs set', function () {
```

Comment:
I would really like to rewrite this so that it isn't tied to the current state of the `testCourse`. We can hopefully use the sync testing utils to construct arbitrary courses here.